packages/auth/src/shared/utils.test.ts

```diff
@@ -0,0 +1,58 @@
+import { IocContract } from '@adonisjs/fold'
+import { Redis } from 'ioredis'
+
+import { AppContext, AppServices } from '../app'
+import { Config } from '../config/app'
+import { createContext } from '../tests/context'
+import { generateApiSignature } from '../tests/apiSignature'
+import { initIocContainer } from '..'
+import { verifyApiSignature } from './utils'
+import { TestContainer, createTestApp } from '../tests/app'
+
+describe('utils', (): void => {
+  let deps: IocContract<AppServices>
+  let appContainer: TestContainer
+  let redis: Redis
+
+  describe('admin api signatures', (): void => {
+    beforeAll(async (): Promise<void> => {
+      deps = initIocContainer({
+        ...Config,
+        adminApiSecret: 'test-secret'
+      })
+      appContainer = await createTestApp(deps)
+      const redis = await deps.use('redis')
```

Comment:
(doesn't fix the `MaxRetriesPerRequestError` error)

```suggestion
     redis = await deps.use('redis')
```